test/output/simple-types.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "element": "boolean",
+    "content": true,
+    "meta": {
+      "id": "boolean"
+    }
+  },
+  {
+    "element": "string",
+    "content": "test",
+    "meta": {
+      "id": "string"
+    }
+  },
+  {
+    "element": "string",
+    "content": "another test",
+    "meta": {
+      "id": "string2"
```

Comment:
Not sure if we shouldn't put a separator between ‘string’ and ‘2’ (e.g. `string.2`, `string-2`).